pkg/kf/commands/service-bindings/integration_test.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package servicebindings
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	. "github.com/google/kf/pkg/kf/testutil"
+)
+
+func TestIntegration_withServiceBroker_Marketplace(t *testing.T) {
```

Comment:
nit: try to make these test names a little more meaningful to what they're testing `TestIntegration_marketplace` or `TestIntegration_vcapInjection` that way when they fail we have a better idea of what is going wrong

Reply:
cleaned up by removing all the "withBlah" parts of the test names